apps/web/app/(use-page-wrapper)/(main-nav)/bookings/[status]/page.tsx

```diff
@@ -33,9 +34,26 @@ const Page = async ({ params }: PageProps) => {
   const t = await getTranslate();
   const session = await getServerSession({ req: buildLegacyRequest(await headers(), await cookies()) });
 
+  let canReadOthersBookings = false;
+  if (session?.user?.id) {
+    const permissionService = new PermissionCheckService();
+    const userId = session.user.id;
+
+    const teamIdsWithPermission = await permissionService.getTeamIdsWithPermission(userId, "booking.read");
+    // We check if teamIdsWithPermission.length > 0.
+    // While this may not be entirely accurate, it's acceptable
+    // because we perform a thorough validation on the server side for the actual filter values.
+    // This variable is primarily for UI purposes.
+    canReadOthersBookings = teamIdsWithPermission.length > 0;
```

Comment:
We need to keep in mind fallback roles here - getTeamIdsWithpermissions doesnt account for that

Reply:
Ah !